src/main/java/com/google/cloud/tools/managedcloudsdk/install/Installer.java

```diff
@@ -57,7 +61,10 @@ public void install()
     command.add("--quiet"); // don't accept user input during install
     command.add("--usage-reporting=" + usageReporting); // usage reporing passthrough
 
-    commandRunner.run(command, installedSdkRoot, null, messageListener);
+    progressListener.update("Installing Cloud SDK");
+    progressListener.update(250);
```

Comment:
It looks like `progressListener` is initialized to start work of 300 in `SdkInstaller#install`. It might be good to add a comment here for where this number comes from or to have all the updates be done in the same scope as where the `totalWork` is set. (Using a different ProgressListener than one set to totalWork of 300 may cause unexpected behavior)

Reply:
This demonstrates why Eclipse has sub-monitors: you're scattering information across the implementation.  The Eclipse convention is that `start()`, `update()`, and `done()` must all be done in the same method.

But it's easy to create a `SubProgressListener` and give it some amount of work.  When its `update()` is called, it just calls update() on its parent.  On `done()`, it calls `update()` with whatever work remains.

Reply:
That's reasonable, I was trying to keep the implementer side simple. But I can do that.